frontend/src/global_styles/openproject/_forms.sass

```diff
@@ -76,7 +76,7 @@ $select-arrow: true !default
   border: $input-border
   border-radius: 2px
   background-color: transparent
-  color: var(--body-font-color)
+  color: var(--body-font-color) !important
```

Comment:
Why is that `important` needed here? Should we rather change the place where it gets overriden?

Reply:
In (webpack:///node_modules/@ng-select/ng-select/themes/default.theme.css), default color of input is #000 so the value that we set for inputs is ignored, so I added !important for the color that we set for inputs and force input value to use body-font-color.

Reply:
But you should be able to override that with a more specific selector in the `autocomplete.sass`. We already have a selector for the input there.